src/AlgebraicGeometry/ToricVarieties/cohomCalg/special_attributes.jl

```diff
@@ -258,11 +258,11 @@ julia> sheaf_cohomology(toric_line_bundle(dP3, [4, 1, 1, 1]), 0)
 ```
 """
 function sheaf_cohomology(l::ToricLineBundle, i::Int; algorithm::String="cohomCalg")
-  if has_attribute(l, :sheaf_cohomology) 
-    return ((get_attribute(l, :sheaf_cohomology)::Vector{ZZRingElem})[i+1])::ZZRingElem
+  if has_attribute(l, :sheaf_cohomology)
+    return ((get_attribute(l, :sheaf_cohomology)::Vector{ZZRingElem})[i + 1])::ZZRingElem
   end
   table = get_attribute!(l, :sheaf_cohomology_table) do
-    Dict{Int, ZZRingElem}()
+    Dict{Int,ZZRingElem}()
```

Comment:
Just to "mimimi" a little: I think not leaving a whitespace after a comma is not good style. But if people decided on that, let's stick with it.